src/integrationTest/java/org/opensearch/security/http/JwtAuthenticationTests.java

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
Should the files in `integrationTest` be added to 2.x? @peternied thoughts?

Reply:
We should have integrationTests for 2.x line.

Reply:
Are you able to run them? iirc there is more to the setup than backporting individual files in `src/integrationTest`?